warehouse/packaging/search.py

```diff
@@ -60,23 +58,8 @@ def from_db(cls, release):
         obj = cls(meta={"id": release.project.normalized_name})
         obj["name"] = release.project.name
         obj["normalized_name"] = release.project.normalized_name
-        obj["version"] = [
-            r.version
-            for r in sorted(
-                release.project.releases,
-                key=lambda r: parse_version(r.version),
-                reverse=True,
-            )
-        ]
-        obj["latest_version"] = first(
-            sorted(
-                release.project.releases,
-                key=lambda r: parse_version(r.version),
-                reverse=True,
-            ),
-            key=lambda r: not r.is_prerelease,
-            default=release.project.releases[0],
-        ).version
+        obj["version"] = [r.version for r in release.project.all_releases]
+        obj["latest_version"] = release.project.latest_release.version
```

Comment:
I forget why I did this, but is the latest version not already the one being passed into this method as ``release``? If it is then this is redundant (and with these changes, causing an additional query to be executed).

Reply:
previously it skipped prereleases, so I kept that logic in place.

Reply:
Hmm, that raises a bit of a point, should we use the *other* data from a pre-release here? Is it confusing to have say the summary and readme from a pre-release being shown with the latest non pre-release version?

I think maybe the version selection logic for indexing should use the same nullslast, order by logic that I posted above that will give the first non-prerelease, unless there is only pre-releases.

Reply:
works for me!

Reply:
addressed.